Fundraising/FundraisingProgress_ReadMe.md

```diff
@@ -1,12 +1,59 @@
-## Fundraising Progress
+![Kingdom First Solutions](https://user-images.githubusercontent.com/81330042/115049197-12a9b580-9ea0-11eb-8e4d-81873a5cd366.png)
 
-*Tested/Supported Rock version: 9.0-10.2*
-*Last Modified: 6/11/2020*
 
-Modified Fundraising Progress core block with every option configurable via block settings. With some additional features including display of the total raised, ability to show an excel export option and select a group instead of just using the GroupId page parameter.
+# Fundraising Progress
+_Tested/Supported in Rock Version:  8.0-12.0_    
+_Released:  6/18/2020_   
+_Updated:  7/7/2020_   
 
-**Display:**
-![](../.screenshots/fundraising_FundraisingProgress_display.png)
+## Summary
+
+This block is an advanced version of the Fundraising Progress block. It gives you the ability to show or hide totals raised, goals and progress for both the entire group and group members. You also have the option to add an Export to Excel button.
+
+Quick Links:
+
+- [What's New](#whats-new)
+- [Configuration](#configuration)
+- [System Email](#system-email)
```

Comment:
@SarahCKopf there is a link in the quick links section to "System Email" but I do not see a section titled that in this readme, is it missing? Can you double check please?

Reply:
@nateh777, When I look at the PDF, there is no system email section there.   Would you like me to take that out?

Reply:
@KelleyLangkamp is our fundraising documentation missing something about the system email? If not, that is fine, we'll just remove this from the quick links. Just something I noticed when reviewing.

Reply:
@SarahCKopf Yes, remove the system email thing. That is a separate package.